lib/live_view_native_swift_ui/modifiers/text_input_and_output/keyboard_shortcut.ex

```diff
@@ -0,0 +1,11 @@
+defmodule LiveViewNativeSwiftUi.Modifiers.KeyboardShortcut do
+  use LiveViewNativePlatform.Modifier
+
+  alias LiveViewNativeSwiftUi.Types.EventModifier
+  alias LiveViewNativeSwiftUi.Types.KeyEquivalent
+
+  modifier_schema "keyboard_shortcut" do
+    field :key, KeyEquivalent
+    field :event_modifiers, EventModifier
```

Comment:
This doesn't match the Swift code.
```suggestion
    field :modifiers, EventModifier
```